survey/4-ai.yml

```diff
@@ -17,6 +17,7 @@ questions:
       - Helped me to become more productive (10% to 30%)
       - I am now very productive (30% to 50%)
       - I am a 10x engineer now (+50%)
+      - It decreased my productivity
```

Comment:
These last 2 are a bit overlapping! We can maybe only keep the last one

Reply:
you mean those two 
```
 - I am a 10x engineer now (+50%)
 - It decreased my productivity
```
i don't think so